src/services/lnd/schema.ts

```diff
@@ -0,0 +1,58 @@
+import * as mongoose from "mongoose"
+
+const Schema = mongoose.Schema
+
+const confirmedDetailsSchema = new Schema<LnPaymentConfirmedDetails>({
+  confirmedAt: {
+    type: Date,
+    required: true,
+  },
+  destination: {
+    type: String,
+    required: true,
+  },
+  revealedPreImage: {
+    type: String,
+    required: true,
+  },
+  roundedUpFee: {
+    type: Number,
+    required: true,
+  },
+  milliSatsFee: {
+    type: Number,
+    required: true,
+  },
+  hopPubkeys: [String],
+})
+
+const paymentAttemptSchema = Schema.Types.Mixed
+
+const paymentSchema = new Schema<LnPaymentType>({
+  createdAt: Date,
```

Comment:
needs to set a default value here. this is not been populated from the demo-success-pre.json from your example

Reply:
I'm not sure if there's value in this here. The intention is that `created_at` gets set from the value in lnd, and before that (`isCompleteRecord: false`) these objects aren't meant to be used but are only a marker with the bare info needed to query lnd and complete when the cronjob is run.

There's other fields we can also infer from the paymentRequest that I deliberately left out as well. I really only included the `paymentHash` as an index and the `paymentRequest` because we don't always get this back from lnd.

Reply:
the value I see in having the createdAt set at this time, is for us to calculate how long a payment is taking to complete (time-to-payment-completion is a key metrics to follow).

maybe we're getting this from lnd?

Reply:
Exactly, the property itself is definitely valuable. What I'm saying is that we always get it from lnd, so there probably isn't value in trying to pre-emptively set it ourselves to always have it overwritten.